src/main/kotlin/com/esops/service/OrderService.kt

```diff
@@ -198,19 +193,8 @@ class OrderService(private var userService: UserService,
 
     fun orderHistory(username: String): List<Order> {
         userService.testUser(username)
-        if (orders[username].isNullOrEmpty()) initializeOrderMapForUser(username)
-        return orders[username]!!.values.toList()
-    }
-
-    private fun initializeOrderMapForUser(username: String) {
-        orders[username] = HashMap()
-    }
-
-    fun clearOrders() {
-        orders = HashMap()
-        orderIDCounter = 0
-        buyOrderQueue.clear()
-        sellOrderQueue.clear()
+        orderRepository.initializeOrderMapIfEmpty(username)
```

Comment:
same. This is not a service responsibility

Reply:
moved to repository